themes/classic/templates/checkout/_partials/cart-voucher.tpl

```diff
@@ -77,7 +72,11 @@
           {/foreach}
           </ul>
         {/if}
```

Comment:
Please indent the `{if}`, the `{/if}` and its content accordingly

Reply:
I just indent it